Zend/tests/add_002.phpt

```diff
@@ -20,10 +20,14 @@ var_dump($c);
 echo "Done\n";
 ?>
 --EXPECTF--
+Notice: You have to implement the __add function in class stdClass to use this operator with an object in %sadd_002.php on line %d
```

Comment:
The notice doesn't make sense for non-user defined classes!
I see no point in emitting it here.

Reply:
I agree that *this* message does not make sense here, but I'm fine with having a notice on object to int conversion (which is [undefined behavior](https://www.php.net/manual/en/language.types.integer.php#language.types.integer.casting.from-other)) here.